spec/models/user_group_spec.rb

```diff
@@ -1,22 +1,67 @@
 require 'rails_helper'
 
 RSpec.describe UserGroup, :type => :model do
-  context 'with groceries' do
+  describe 'group groceries' do
+    context 'with groceries' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.